src/Symfony/Bridge/PhpUnit/Tests/DeprecationErrorHandler/ConfigurationTest.php

```diff
@@ -17,6 +17,9 @@
 use Symfony\Bridge\PhpUnit\DeprecationErrorHandler\DeprecationGroup;
 use Symfony\Component\ErrorHandler\DebugClassLoader;
 
+/**
+ * @requires PHPUnit < 10
```

Comment:
shouldn't this have the attribute as well, so that PHPUnit 11 excludes it ?

Reply:
in fact, I even think that having the attribute without the annotation should be sufficient